main.go

```diff
@@ -110,9 +111,23 @@ const (
 func addLogsHandler(mux *http.ServeMux, logs *supervisord.Logs) {
 	l := logrus.WithField("component", "logs.zip")
 
+	cfg := config.NewService()
+	if err := cfg.Load(); err != nil {
+		l.Panicf("Failed to load config: %+v", err)
+	}
+
 	mux.HandleFunc("/logs.zip", func(rw http.ResponseWriter, req *http.Request) {
+		contextTimeout := 10 * time.Second
+		// increase context timeout if pprof query parameter exist in request
+		pprofQueryParameter, _ := strconv.Atoi(req.FormValue("pprof"))
```

Comment:
I suggest using `strconv.ParseBool` instead of `strconv.Atoi`.
Do not ignore err, let's at least log the message.

Reply:
Changed.